apps/studio/components/interfaces/DataWarehouse/FormFooterChangeBadge.tsx

```diff
@@ -0,0 +1,43 @@
+import { AnimatePresence, motion } from 'framer-motion'
+import { Badge } from 'ui'
+
+interface FormFooterChangeBadgeProps {
+  formState: {
+    dirtyFields: Record<string, boolean>
+  }
+}
+export const FormFooterChangeBadge = ({ formState }: FormFooterChangeBadgeProps) => {
```

Comment:
Is this actually used anywhere in this PR?

Reply:
yup! DiskManagementPanelForm L566